ImageEditor/src/main/kotlin/org/wordpress/android/imageeditor/preview/PreviewImageFragment.kt

```diff
@@ -98,4 +105,15 @@ class PreviewImageFragment : Fragment() {
             }
         )
     }
+
+    private fun startUCrop(inputFilePath: String) {
+        findNavController().navigate(
+            R.id.action_previewFragment_to_ucropFragment,
+            UCropUtil.getUCropOptionsBundle(
+                File(inputFilePath),
+                // TODO: output file location
```

Comment:
I think the cacheDir is an ok location or do you have any concerns?

Reply:
No concerns. Just wanted to confirm if it is ok.